src/buildHTML.js

```diff
@@ -206,166 +206,13 @@ export const isLeftTight = function(node) {
     return node.hasClass("mtight");
```

Comment:
I think we can remove above warning, too.

Reply:
Done.